modules/mobile/app/serializers/mobile/v0/military_information_serializer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'fast_jsonapi'
+
+module Mobile
+  module V0
+    class MilitaryInformationSerializer
+      include FastJsonapi::ObjectSerializer
+      set_key_transform :camel_lower
```

Comment:
I think you should be able to remove this now that the `iam_headers` include the olive branch key inflection

Reply:
Was this comment addressed and ready for review?

Reply:
Yup, it was removed as requested.